### ISSUE TITLE
Any way to reload tooltips on a specific ID?

### ISSUE THREAD

**Issue by Dillybob92** [darsain/tooltips#11]
For example, to toggle a tooltip you must reload and toggle the tooltip:

'tips.reload();
tips.toggle(qs('[item_id="{$itemid}"]'));'

Is there anyway to do just reload the tooltips on one element instead of all of them? The reason I'm asking is because for my websocket server when someone enters the channel their tooltip needs to be added dynamically and reloaded. But I don't want to call reload() every time because someone can be viewing their items in their inventory, etc. (And then their tooltip will disappear because someone joined the channel lol)

For example: I want to do something like this:
qs('[user_id="'+cb.user_id+'"]').tips.reload();

**Comment by Dillybob92** [comment]
TLDR:

we can do .remove(element) or .toggle(element) but we cannot do it on .reload(). There is o no (element) after .reload. Is it possible to add that functionality?  .reload(element) is what I'm trying to do afaik, hope this helps :D

**Comment by Dillybob92** [comment]
Nevermind. There is a bug when using the data selector elements though.

I switched to using just a regular ID from an element and used .add and it works! 

'tips.add(qs('#characterID'+characterid+''));'

**Comment by darsain** [comment]
You can't just continuously call `.add()`, you are causing a memory leak. You have to also call `.remove()` on removed elements.

But I'm  glad it works for  you.

**Comment by Dillybob92** [comment]
Yeah,  on my socket.onclose = function I am using remove. I accidentally forgot to do that, and was testing people leaving/entering my channels and under inspect element on chrome I saw the memory leak happens and the div floating there lol.

By the way, after using jsbeautify and looking at your source code. Are you on a different level of JavaScript? Because I don't understand anything in it. I guess I am obviously not as smart as you, but gee willy man, it's like javascript 2.0.
